core/src/main/java/org/apache/carbondata/core/metadata/schema/BucketingInfo.java

```diff
@@ -44,4 +51,21 @@ public int getNumberOfBuckets() {
     return numberOfBuckets;
   }
 
+  @Override public void write(DataOutput out) throws IOException {
+    out.writeInt(numberOfBuckets);
+    out.writeInt(listOfColumns.size());
+    for (ColumnSchema aColSchema : listOfColumns) {
+      aColSchema.write(out);
+    }
+  }
+
+  @Override public void readFields(DataInput in) throws IOException {
+    this.numberOfBuckets = in.readInt();
+    int colSchemaSize = in.readInt();
+    listOfColumns = new ArrayList<>(colSchemaSize);
```

Comment:
handled.It would not make a difference though,No local variable with same name to cause confusion